### DISCUSSION TITLE
Red border appeared around window

### DISCUSSION THREAD

**Discussion by jam1015** [tridactyl/tridactyl#3736] [Q&A]
First off, let me say thank you so much (if you're reading this) for developing such a nice extension.  Really adds productivity and ease of use once you get the hang of it and feels more vim like than the other extensions I briefly tried. 

Ok what I'm asking about: I briefly messed up typing, and at some point a red border appeared around the Firefox window.  I'm trying to reproduce it but I can't. Does anyone know what this might have been?  I am using the dark colorscheme.

It could have been a different extension I have that caused it but am wondering if anyone might know.

**Comment by bovine3dom** [comment]
Thanks for your kind words about Tridactyl : )

I have no idea where your red border came from. I don't think it's something we have in Tridactyl. Sorry that I can't be more help!
